arrows/mvg/applets/tests/data/init_cameras_testing.conf

```diff
@@ -0,0 +1,13 @@
+# Default configuration for the init_cameras_landmarks applet
```

Comment:
This filename is `init_cameras_testing.conf` but the test command is trying to load `init_cameras_landmarks_testing.conf` we should fix the name of this file.